roadmap/implementers-guide/src/node/disputes/README.md

```diff
@@ -1,40 +1,17 @@
-
-This is responsible for tracking and initiating disputes. Disputes will be initiated either externally by another subsystem which has identified an issue with a parablock or internally upon observing two statements which conflict with each other on the validity of a parablock.
-
-No more than one statement by each validator on each side of the dispute needs to be stored. That is, validators are allowed to participate on both sides of the dispute, although we won't write code to do so. Such behavior has negative extractable value in the runtime.
-
-This will notify the dispute participation subsystem of a new dispute if the local validator has not issued any statements on the disputed candidate already.
-
-Locally authored statements related to disputes will be forwarded to the dispute distribution subsystem.
-
-This subsystem also provides two further behaviors for the interactions between disputes and fork-choice
-    - Enhancing the finality voting rule. Given description of a chain and candidates included at different heights in that chain, it returns the `BlockHash` corresponding to the highest `BlockNumber` that there are no disputes before. I expect that we will slightly change `ApprovalVoting::ApprovedAncestor` to return this set and then the target block to vote on will be further constrained by this function.
-    - Chain roll-backs. Whenever importing new blocks, the header should be scanned for a roll-back digest. If there is one, the chain should be rolled back according to the digest. I expect this would be implemented with a `ChainApi` function and possibly an `ApprovalVoting` function to clean up the approval voting DB.
-
-### Dispute Participation
-
-This subsystem ties together the dispute tracker, availability recovery, candidate validation, and dispute distribution subsystems. When notified of a new dispute by the Dispute Tracker, the data should be recovered, the validation code loaded from the relay chain, and the candidate is executed.
-
-A statement depending on the outcome of the execution is produced, signed, and imported to the dispute tracker.
-
-### Dispute Distribution
-
-This is a networking component by which validators notify each other of live disputes and statements on those disputes.
-
-Validators will in the future distribute votes to each other via the network, but at the moment learn of disputes just from watching the chain.
+If approval voting finds an invalid candidate, a dispute is raised. The disputes
+subsystems make that possible:
+
+1. Disputes can be raised
+2. Disputes (votes) get propagated to all other validators
+3. Votes get recorded as necessary
+3. Nodes will participate in disputes in a sensible fashion
+4. Finality is halted if a candidate on the chain is disputed
+5. Chains can be reverted in case a dispute concluded invalid
+6. Votes are provided to the provisioner so losing validators can be slashed on chain
```

Comment:
I'd say something like
6. Votes are being imported on chain via the provisioner
7. Once a dispute concludes on chain, validators on the losing side can be slashed

Reply:
I compressed it a bit, as this is already outside the scope of the dispute subsystem's, in particular it should not be its own point.

Reply:
Makes sense. I was wondering if slashing is the only reason to put them onchain. I think we want that also to reach consensus on what's concluded so we can revert on invalid disputes safely (and not revert only on some validators).